pcsx2/gui/GlobalCommands.cpp

```diff
@@ -173,26 +174,26 @@ namespace Implementations
 	void GSwindow_CycleAspectRatio()
 	{
 		AspectRatioType& art = g_Conf->GSWindow.AspectRatio;
-		wxString arts(L"Not modified");
+		const char *arts = "Not modified";
```

Comment:
Didn't we have a discussion earlier that the asterisk should be placed near the data type instead of the variable name?